src/main/java/productDependency/CreateDependentCategoryPage.java

```diff
@@ -0,0 +1,974 @@
+		Select se = new Select(Productelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 7, 1));
+
+	}
+	
+	//*[@id="product.dependencyMin"]
+		@FindBy(how = How.XPATH, using = "//*[@id='product.dependencyMin']")
+		private WebElement enterMin1;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterMin1() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Min value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterMin);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			enterMin1.clear();
+			enterMin.sendKeys("1");
+		}
+		
+		@FindBy(how = How.XPATH, using = "//*[@id='product.dependencyMax']")
+		private WebElement enterMax1;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterMax1() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Max value");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterMax1);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			enterMax1.clear();
+			enterMax1.sendKeys("5");
+		}
+		
+		@FindBy(how = How.XPATH, using = "//input[@name='product.descriptions[0].content']")
+		private WebElement enterEnglishDescription2;
+
+		/**
+		 * Method to Enter English Description.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterEnglishDescription2() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter English Description");
+			Assert.assertTrue(enterEnglishDescription.isDisplayed());
+			enterEnglishDescription.sendKeys(BasePage.getCellData(xlsxName, sheetName, 4, 5));
+
+		}
+
+		@FindBy(how = How.XPATH, using = "//input[@name='product.number']")
+		private WebElement enterProductCode2;
+
+		/**
+		 * Method to Enter Product Code.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterProductCode2() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Product Code");
+			Assert.assertTrue(enterProductCode.isDisplayed());
+			enterProductCode.sendKeys(BasePage.getCellData(xlsxName, sheetName, 4, 5));
+
+		}
+		
+		@FindBy(how = How.XPATH, using = "//input[@name='product.rate']")
+		private WebElement enterPriceRate2;
+
+		/**
+		 * Method to Enter Price Rate.
+		 * 
+		 * @throws IOException
+		 */
+		public void enterPriceRate2() throws IOException {
+			CreateDependentCategoryPage sp = new CreateDependentCategoryPage(driver);
+			log.info("Enter Price Rate");
+			JavaScriptExec.sleep();
+			WaitClass.WaitForElementisDisplay(driver, 10, enterPriceRate);
+			Assert.assertTrue(enterPriceRate.isDisplayed());
+			//enterPriceRate1.clear();
+			enterPriceRate.sendKeys(BasePage.getCellData(xlsxName, sheetName, 6, 5));
+		}
```

Comment:
@mayankchhajer remove this extra WAIT and commented line

Reply:
Done